core/src/main/groovy/com/novoda/gradle/release/AndroidLibrary.groovy

```diff
@@ -5,13 +5,14 @@ import org.gradle.api.Project
 import org.gradle.api.artifacts.Dependency
 import org.gradle.api.artifacts.ModuleDependency
 import org.gradle.api.artifacts.PublishArtifact
+import org.gradle.api.attributes.Usage
 import org.gradle.api.internal.DefaultDomainObjectSet
 import org.gradle.api.internal.component.SoftwareComponentInternal
-import org.gradle.api.internal.component.Usage
+import org.gradle.api.internal.component.UsageContext
 
 class AndroidLibrary implements SoftwareComponentInternal {
 
-    private final Usage runtimeUsage
+    private final UsageContext runtimeUsage
```

Comment:
@tasomaniac I just update issue #112. I don't think we have other option than re-release for 3.4+ only. More details can be found [here](https://github.com/novoda/bintray-release/issues/112#issuecomment-293674149).